.gitignore

```diff
@@ -12,3 +12,6 @@
 hs_err_pid*
 /target/
 /bin/
+/cql2pgjson-library/target/
+/cql2pgjson-cli/target/
+/cql2pgjson/target/
```

Comment:
Can you also add `.*.swp` to .gitignore?

Reply:
And delete `cql2pgjson/.pom.xml.swp`?

Reply:
Regarding the .gitignore part, in my opinion it is better for each developer
to add settings that are specific to their own system to their local ~/.gitignore_global file (or whatever is specified by their core.excludesFile config). Rather than adding to every project's .gitignore file.
See https://dev.folio.org/guides/developer-setup/#use-gitignore